src/controller/java/generated/java/matter/devicecontroller/cluster/clusters/AccessControlCluster.kt

```diff
@@ -37,144 +44,542 @@ class AccessControlCluster(
   class AttributeListAttribute(val value: List<UInt>)
 
   suspend fun readAclAttribute(): AclAttribute {
-    // Implementation needs to be added here
-  }
+    val ATTRIBUTE_ID_ACL: UInt = 0u
```

Comment:
do we need a named suffix here? given that I am inside `readAclAttribute` it seems clear that the attribute ID is ACL. Would it make sense to just name this `ATTRIBUTE_ID` ? may simplify some jinja code to not need to suffix things.